service/src/main/java/com/zebrunner/reporting/service/ConfigurationService.java

```diff
@@ -71,6 +71,6 @@ public ConfigurationService(
     }
 
     private boolean isSlackAvailable() {
-        return slackService.isEnabledAndConnected(null) && slackService.getWebhook() != null;
+        return notificationService.isEnabledAndConnected(null) && notificationService.getWebhook(null) != null;
```

Comment:
We have to adjust this since now we have more than just default integration. Obtain slack integration and check if it's connected
P.S.: Why do we need this in configuration service at all?

Reply:
I don't know why do we need this service)